### PR TITLE
Update Giant Swarm website URL

### PR DESCRIPTION
**Pull request by marians**
The link shown currently results in a redirect. I'd prefer to have the actual URL in the docs. Thanks!

### CODE REVIEW

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ The following production environment solutions table lists the providers and the
 | [Fedora (Multi Node)](https://kubernetes.io/docs/getting-started-guides/fedora/flannel_multi_node_cluster/)     |      |  |      | |    &#x2714;       | &#x2714;
 | [Fedora (Single Node)](https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/)     |      |  |      | |           | &#x2714;
 | [Gardener](https://gardener.cloud/) | &#x2714; | &#x2714; | &#x2714; | &#x2714; | &#x2714; | [Custom Extensions](https://github.com/gardener/gardener/blob/master/docs/extensions/overview.md) |
-| [Giant Swarm](https://giantswarm.io/)     | &#x2714;       | &#x2714; |   &#x2714;    |     |
```
Comment (RA489) [inline content/en/docs/setup/_index.md]: Both the link pointing to the same page.

File: content/en/docs/setup/_index.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ The following production environment solutions table lists the providers and the
...
+| [Giant Swarm](https://www.giantswarm.io/)     | &#x2714;       | &#x2714; |   &#x2714;    |     |
```
Comment (RA489) [inline content/en/docs/setup/_index.md]: this one too pointing to same page. Is their any specific purpose for doing this change?

Comment (marians) [reply content/en/docs/setup/_index.md]: The actual website URL is https://www.giantswarm.io/, so I am trying to save a redirect where it's not necessary. General web hygiene.

**Pull request merged**